datastore/admin/datastore_admin_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package samples
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestCreate(t *testing.T) {
+	client, err := clientCreate(ioutil.Discard)
+	if err != nil {
+		t.Fatalf("clientCreate: %v", err)
+	}
+	defer client.Close()
+}
+
+func TestIndexList(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	if _, err := indexList(ioutil.Discard, tc.ProjectID); err != nil {
+		t.Fatalf("indexList: %v", err)
+	}
+}
+
+func TestIndexGet(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	// Get first index from the list of indexes.
+	indices, _ := indexList(ioutil.Discard, tc.ProjectID)
+	indexID := indices[0].IndexId
+	if len(indices) == 0 {
+		t.Skip("Skipping datastore test. At least one index should present in database.")
```

Comment:
Should we combine combine these tests into a single test? That way, we could create the index then get it in a single test.

Reply:
Thanks, good idea

Reply:
Could we also merge `TestCreate` in here? Is the create operation guaranteed to be faster?

Reply:
Practically, It takes the same time. Ok, I'll merge it too

Reply:
If we create an index above, then `indexList` returns an empty slice, the test should fail, right? Sorry I was unclear -- I meant is list guaranteed to contain the index that was just created? We might need a retry if not.

Reply:
Sorry, I'm not sure, I'm understand you correctly.  We aren't creating index here, 'cause `Create` method for index not implemented yet. I could now only take an index from the project. In this case I need a retry too?

Reply:
Oh, sorry! Yeah, that makes sense. No, I don't think we need to add a retry. This looks good.